src/components/MobileNavbar/MobileNavbar.js

```diff
@@ -0,0 +1,183 @@
+import React, { useState } from 'react'
+import { withRouter, NavLink as Link } from 'react-router-dom'
+import cx from 'classnames'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { Button } from '@santiment-network/ui'
+import MobileNavbarAction from './MobileNavbarAction'
+import styles from './MobileNavbar.module.scss'
+import * as actions from './../../actions/types'
+
+const NAVBAR_LINKS = [
+  {
+    link: '/trends',
+    label: 'Trends',
+    linkTo: '/labs/trends',
+    iconType: 'fire'
+  },
+  {
+    link: '/assets',
+    label: 'Assets',
+    linkTo: '/assets/all',
+    iconType: 'assets'
+  },
+  {
+    link: '/insights',
+    label: 'Insights',
+    iconType: 'bulb',
+    linkTo: '/insights'
+  }
+]
+
+const MENU_LINKS = [
+  { link: '/account', label: 'Account settings' },
+  { link: '/support', label: 'Support' }
+]
+
+const MobileNavbar = ({ history, isLogined, activeLink, logout }) => {
+  const [isOpened, setOpened] = useState(true)
+
+  const toggleMenu = () => setOpened(!isOpened)
+
+  const handleNavigation = linkTo => {
+    isOpened && toggleMenu()
+    history.push(linkTo)
+  }
+
+  return (
+    <div className={cx({ [styles.overlay]: isOpened })}>
+      <div className={styles.wrapper}>
+        {NAVBAR_LINKS.map(({ link, label, linkTo, iconType }) => {
+          return (
+            <MobileNavbarAction
+              key={label}
+              isActive={!isOpened && activeLink.includes(link)}
+              linkTo={linkTo}
+              iconType={iconType}
+              onClick={handleNavigation}
+              label={label}
+            />
+          )
+        })}
+        <MobileNavbarAction
+          onClick={toggleMenu}
+          iconType={isOpened ? 'close' : 'hamburger'}
```

Comment:
`iconType` could be calculated inside `MobileNavbarAction` via `isActive`. @Partysun What do you think?

Reply:
@DmitriVanGuard yes, but than MobileNavbarAction will be more specific.
We can possibly to move it to san-ui later.